freshgrub-registration/src/main/java/com/example/demo/dto/LoginDTO.java

```diff
@@ -0,0 +1,41 @@
+package com.example.demo.dto;
```

Comment:
This is good . But there's something called Lombok .
which i would want you guys to research on.
Its very easy.